image/image_test.go

```diff
@@ -72,13 +74,10 @@ type imageSuite struct {
 	stdout *bytes.Buffer
 	stderr *bytes.Buffer
 
-	downloadedSnaps map[string]string
-	storeSnapInfo   map[string]*snap.Info
-	storeActions    []*store.SnapAction
-	tsto            *image.ToolingStore
+	storeActions []*store.SnapAction
+	tsto         *image.ToolingStore
 
-	storeSigning *assertstest.StoreStack
-	brands       *assertstest.SigningAccounts
+	seedtest.SeedSnaps
```

Comment:
Given how important this is now I wonder if a small comment would make sense here? Like `// this will bring important helpers like "SetupAssertSigning" and "MakeAssertedSnap"` ?